data.sql

```diff
@@ -13,3 +13,29 @@ VALUES ('Pikachu', '7-1-2021', '1', 'false', '15.04')
 INSERT INTO animals
 (name, date_of_birth, escape_attempts, neutered, weight_Kg)
 VALUES ('Devimon', '12-5-2017', '5', 'true', '11')
+
+--second batch of data
+
+INSERT INTO animals
+(name, date_of_birth, escape_attempts, neutered, weight_kg)
+VALUES ('Charmander', '2020=02-08', '0', 'false', '11');
+
+INSERT INTO animals
+(name, date_of_birth, escape_attempts, neutered, weight_kg)
+VALUES ('Plantmon', '2022-11-15', '2', 'true', '-5.7');
+
+INSERT INTO animals
+(name, date_of_birth, escape_attempts, neutered, weight_kg)
+VALUES ('Squirtle', '1993-04-02', '3', 'false', '-12.13');
+
+INSERT INTO animals
+(name, date_of_birth, escape_attempts, neutered, weight_kg)
+VALUES ('Angemon', '2005-07-07', '1', 'true', '-45');
```

Comment:
- [ ] The `date_of_birth` for this data is `2005-06-12` instead of `2005-07-07`. Please check the project requirement and make necessary corrections 😺.